projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -0,0 +1,53 @@
+{{- /*
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.alertmanager.enabled }}
+apiVersion: v1
+kind: List
+metadata:
+  resourceVersion: ""
+items:
+  - apiVersion: monitoring.coreos.com/v1alpha1
+    kind: AlertmanagerConfig
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/last-applied-: ""
+      generation: 6
```

Comment:
This also should not be needed. The generation metadata field is managed by Kubernetes.